.github/workflows/test.yml

```diff
@@ -15,14 +15,27 @@ jobs:
         strategy:
             fail-fast: true
             matrix:
-                os: [ubuntu-22.04]  # , ubuntu-24.04]
-                python-version: ['3.10']  # '3.11', '3.12']
-                numpy-version: ['<1.23']
-                astropy-version: ['<6.1', '<7']
+              include:
+                - os: ubuntu-latest
+                  python-version: '3.13'
+                  numpy-version: '<2.3' # Numba does not yet support NumPy 2.3
+                  astropy-version: '<8.0'
+                - os: ubuntu-latest
+                  python-version: '3.12'
+                  numpy-version: '<2.2'
+                  astropy-version: '<7.0'
+                - os: ubuntu-latest
+                  python-version: '3.11'
+                  numpy-version: '<2.1'
+                  astropy-version: '<6.1'
+                - os: ubuntu-latest
+                  python-version: '3.10'
+                  numpy-version: '<2.0'
+                  astropy-version: '<6.1'
 
         env:
-            DESIUTIL_VERSION: 3.4.3
-            DESIMODEL_VERSION: 0.19.2
+            DESIUTIL_VERSION: 3.5.0
```

Comment:
Although it won't make a huge difference for the tests, this is not the latest version of desiutil.

Also, you're continuing to install desiutil from GitHub instead of PyPI. If that's necessary for the test, please add a comment as to why.